.tekton/pipeline-service-downgrade-test.yaml

```diff
@@ -0,0 +1,408 @@
+                  valueFrom:
+                    secretKeyRef:
+                      name: hypershift-bitwarden
+                      key: "BW_CLIENTID"
+                - name: BW_CLIENTSECRET
+                  valueFrom:
+                    secretKeyRef:
+                      name: hypershift-bitwarden
+                      key: "BW_CLIENTSECRET"
+                - name: BW_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      name: hypershift-bitwarden
+                      key: "BW_PASSWORD"
+              command:
+                - $(workspaces.source.path)/ci/images/ci-runner/hack/bin/deploy-cluster.sh
+      - name: setup-ci-runner-container
+        runAfter:
+          - "deploy-cluster"
+        params:
+          - name: target_branch
+            value: $(params.target_branch)
+        workspaces:
+          - name: source
+            workspace: source
+          - name: kubeconfig-dir
+            workspace: upgrade-shared-workspace
+        taskSpec:
+          params:
+            - name: target_branch
+          workspaces:
+            - name: source
+            - name: kubeconfig-dir
+          kind: ClusterTask
+          steps:
+            - name: create-ci-runner-container
+              image: quay.io/redhat-pipeline-service/ci-runner:$(params.target_branch)
+              resources:
+                requests:
+                  memory: 500Mi
+                  cpu: 300m
+              env:
+                - name: KUBECONFIG
+                  value: "$(workspaces.kubeconfig-dir.path)/kubeconfig"
+              script: |
+                $(workspaces.source.path)/ci/images/ci-runner/hack/bin/create-ci-runner-container.sh
+            - name: copy-plnsvc-code
+              image: quay.io/redhat-pipeline-service/ci-runner:$(params.target_branch)
+              resources:
+                requests:
+                  memory: 500Mi
+                  cpu: 300m
+              workingDir: "$(workspaces.source.path)"
+              env:
+                - name: KUBECONFIG
+                  value: "$(workspaces.kubeconfig-dir.path)/kubeconfig"
+              command:
+                - $(workspaces.source.path)/ci/images/ci-runner/hack/bin/copy-plnsvc-code.sh
+      - name: plnsvc-setup
+        runAfter:
+          - "setup-ci-runner-container"
+        workspaces:
+          - name: kubeconfig-dir
+            workspace: upgrade-shared-workspace
+          - name: source
+            workspace: source
+        params:
+          - name: repo_url
+            value: $(params.repo_url)
+          - name: revision
+            value: $(params.revision)
+          - name: target_branch
+            value: $(params.target_branch)
+        taskSpec:
+          workspaces:
+            - name: kubeconfig-dir
+            - name: source
+          params:
+            - name: repo_url
+            - name: revision
+            - name: target_branch
+          kind: ClusterTask
```

Comment:
is there a reason for this task to be a `ClusterTask`?

Reply:
I removed them and lets see the CI results